### PR TITLE
Create jensen bio page

### PR DESCRIPTION
**Pull request by ZihengSun**

### CODE REVIEW

**Comment by JessicaS11** [comment]
@ZihengSun Could you please rename your file to use a dash instead of an underscore (`jensen-sun.yaml`)?

**Comment by ZihengSun** [comment]
> @ZihengSun Could you please rename your file to use a dash instead of an underscore (`jensen-sun.yaml`)?

Oh yeah sure.

**Comment by ZihengSun** [comment]
Done, but the action keeps failing. I am looking into it.

**Comment by ZihengSun** [comment]
My bad. I always has problems with indentation. It passed now.

File: team/jensen-sun.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+- title: Jensen (Ziheng) Sun
+  avatar: https://avatars.githubusercontent.com/u/17322987
+  role: Research Associate Professor
+  organizations:
+    - name: George Mason University
+  bio: I am a workflow guy and practitioner on using AI to address cool Earth scientific questions.
```
Comment (scottyhq) [inline team/jensen-sun.yaml]:
```suggestion
  bio: > 
    I am a workflow guy and practitioner on using AI to address cool Earth scientific questions. 
```

Comment (scottyhq) [reply team/jensen-sun.yaml]: https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines/21699210#21699210

**Review by scottyhq — CHANGES_REQUESTED** [review]
see above to get the bio to render correctly

**Comment by ZihengSun** [comment]
@scottyhq thanks! corrected.

**Comment by ZihengSun** [comment]
The intro page doesn't pop up because I put brackets in my name. Removing ( and ) did the trick :)

**Review by scottyhq — APPROVED** [review]

**Pull request merged**